specification/securityinsights/resource-manager/common/2.0/types.json

```diff
@@ -0,0 +1,77 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
+    "ResourceWithEtag": {
+      "allOf": [
+        {
+          "$ref": "../../../../common-types/resource-management/v3/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "An azure resource object with an Etag property",
+      "properties": {
+        "etag": {
+          "description": "Etag of the azure resource",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    }
+  },
+  "parameters": {
```

Comment:
>parameters [](http://example.com/codeflow?start=3&length=10)

You just added this params but are they used in any of the existing swaggers?

Reply:
@jorgecotillo Not yet. They will be used in my coming PRs (in a new Stable and Preview versions). Wanted to keep my PRs small and simple. 
The main purpose of this PR is removing our Resource definition and referencing the common-types one.

Reply:
Since there's no API update involved, I just removed WaitForArmFeedback label.